Gems/ScriptCanvas/Code/Tools/TranslationGeneration.cpp

```diff
@@ -860,7 +872,7 @@ namespace ScriptCanvasEditorTools
             Helpers::GetTypeNameAndDescription(resultParameter->m_typeId, resultName, resultDescription);
 
             result.m_typeId = resultKey;
-            result.m_details.m_name = resultParameter->m_name;
+            result.m_details.m_name = behaviorMethod->GetArgumentName(0) ? *behaviorMethod->GetArgumentName(0) : resultName;
```

Comment:
Same here. The `GetArgumentName(0)` can be stored in a variable here to avoid multiple function calls.